tests/integration/document_db.py

```diff
@@ -0,0 +1,205 @@
+import logging
+from typing import Callable, List, Optional, Type
+
+import pytest
+
+from pyatlan.client.atlan import AtlanClient
+from pyatlan.model.assets import Asset, Connection, DocumentDBCollection, DocumentDBDatabase
+from pyatlan.model.enums import AtlanConnectorType
+from pyatlan.model.response import A, AssetMutationResponse
+from tests.integration.client import TestId
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="module")
+def upsert(client: AtlanClient):
+    guids: List[str] = []
+
+    def _upsert(asset: Asset) -> AssetMutationResponse:
+        _response = client.asset.save(asset)
+        if (
+            _response
+            and _response.mutated_entities
+            and _response.mutated_entities.CREATE
+        ):
+            guids.append(_response.mutated_entities.CREATE[0].guid)
+        return _response
+
+    yield _upsert
+
+    for guid in reversed(guids):
+        response = client.asset.purge_by_guid(guid)
+        if (
+            not response
+            or not response.mutated_entities
+            or not response.mutated_entities.DELETE
+        ):
+            LOGGER.error(f"Failed to remove asset with GUID {guid}.")
+
+
+def verify_asset_created(response, asset_type: Type[A]):
+    assert response.mutated_entities
+
+def verify_asset_updated(response, asset_type: Type[A]):
+    assert response.mutated_entities
+    assert response.mutated_entities.CREATE is None
+    assert response.mutated_entities.UPDATE
+    assert len(response.mutated_entities.UPDATE) == 1
+    assets = response.assets_updated(asset_type=asset_type)
+    assert len(assets) == 1
+
+class TestConnection:
+    connection: Optional[Connection] = None
+
+    def test_create(
+        self,
+        client: AtlanClient,
+        upsert: Callable[[Asset], AssetMutationResponse],
+    ):
+        role = client.role_cache.get_id_for_name("$admin")
+        assert role
+        connection_name = TestId.make_unique("DOC_Conn")
+        c = Connection.create(
```

Comment:
Use the latest naming - `creator()` (`create()` is deprecated):
```suggestion
        c = Connection.creator(
```